packages/bitcore-wallet-service/src/lib/emailservice.ts

```diff
@@ -93,17 +93,17 @@ export class EmailService {
       });
     };
 
-    this.defaultLanguage = opts.emailOpts.defaultLanguage || 'en';
-    this.defaultUnit = opts.emailOpts.defaultUnit || 'btc';
+    this.defaultLanguage = opts.emailOpts && opts.emailOpts.defaultLanguage ? opts.emailOpts.defaultLanguage : 'en';
```

Comment:
maybe add:

`opts.emailOpts = opts.emailOpts || {};`

 before all this checks, to simplify all the following lines?

Reply:
Yes. Done!